kafka-ui-react-app/src/components/common/Dashboard/MetricsWrapper.tsx

```diff
@@ -4,17 +4,19 @@ import cx from 'classnames';
 interface Props {
   title?: string;
   wrapperClassName?: string;
+  levelClassName?: string;
```

Comment:
I would suggest to add multiline attr instead
```suggestion
  multiline?: boolean;
```
or 
```suggestion
  levelClassName?: 'level-multiline';
```

Reply:
Done👌